### DISCUSSION TITLE
Include null/blank values in responses

### DISCUSSION THREAD

**Discussion by ericwilhite** [hasura/graphql-engine#7275] [Question]
When querying with the admin header the responses I receive back include all fields described in the query, including null/blank values.

When querying with a user's api key (via auth hook) the null/blank values are stripped from the return response.

How can I configure Hasura, my query, or the user auth system to include these?

**Comment by motleydev** [comment]
Hi @ericwilhite can you share some details about the hook you are using? What are the services being used? Thanks!
